_assets/ci/Jenkinsfile.android

```diff
@@ -59,8 +58,8 @@ pipeline {
 
     stage('Vendoring check') { steps { dir(env.STATUS_PATH) {
       /* fail build if vendoring hasn't been done */
-      sh 'GO111MODULE=on make vendor'
-      sh 'git diff --exit-code --no-color --stat vendor/'
+      //sh 'GO111MODULE=on make vendor'
+      //sh 'git diff --exit-code --no-color --stat vendor/'
     } } }
```

Comment:
It is good that this fixes the issue, I suppose this build would just crash if a future PR hasn't done the vendoring. ...

Reply:
Yeah. But that's not a proper fix. The `gomobile bind` command appears to depend on the `bind` module, not the `gobind` command, which is why it wants `go get golang.org/x/mobile`, but that changes `go.mod`, and running `make vendor` reverts that.

I have no idea what the proper fix for this is.

Reply:
I have an idea. I'll use `GO111MODULE=off` but only with the `go get golang.org/x/mobile` line.

Reply:
Ah, but that won't work...
```
GO111MODULE=off go get golang.org/x/mobile@5d9a3325
go: can only use path@version syntax with 'go get' and 'go install' in module-aware mode
make: *** [Makefile:244: gomobile-install] Error 1
```
For fuck's sake...

Reply:
These issues have been also reported here: https://github.com/golang/go/issues/46943

Reply:
Okay, I think I got it, and the secret sauce is this:
```Makefile
install-gomobile: ##@install Go Mobile Build Tools
	GO111MODULE=on  go install golang.org/x/mobile/cmd/...@5d9a3325
	GO111MODULE=on  go mod download golang.org/x/exp@ec7cb31e
	GO111MODULE=off go get -d golang.org/x/mobile/cmd/gobind
```